lib/hammer_cli_katello/composite_content_view_name_resolvable.rb

```diff
@@ -10,15 +10,13 @@ def composite_content_view_resolve_options(options)
       }
     end
 
-    def all_options
-      result = super.clone
-      if result['option_composite_content_view_name'] &&
-         result['option_composite_content_view_id'].nil?
-        options = composite_content_view_resolve_options(result)
-        result['option_composite_content_view_id'] = resolver.content_view_id(options)
-        @all_options = result
+    def options
+      if super['option_composite_content_view_name'] &&
```

Comment:
I'm trying to modify the instance variable @options or @all_options in the [option_collector](https://github.com/theforeman/hammer-cli/blob/master/lib/hammer_cli/options/option_collector.rb#L12). The methods use the `||=` syntax to cache the result, so it's not bad performance-wise.